### PR TITLE
UnitTesting class

### PR DESCRIPTION
**Pull request by Tolyaspit**
Please check changes and approve it)

### CODE REVIEW

**Comment by Tolyaspit** [comment]
There are moments where I had duplicate couple times the "List<NewTicket> result = ticketsService.getAllTickets();". I'm not sure if I need to make it as a separate class in there. If you have some advice to avoid it , please let me know :)

File: src/test/java/com/andersen/course/service/TicketsServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+class TicketsServiceTest {
```
Comment (Alexmansar) [inline src/test/java/com/andersen/course/service/TicketsServiceTest.java:22]: you can also add here **@'ExtendWith(MockitoExtension.class)**
and remove 
```
 @BeforeEach
    void setUp(){
        MockitoAnnotations.openMocks(this);
    }
```

File: src/test/java/com/andersen/course/service/TicketsServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    private TicketsService ticketsService;
+
+    @BeforeEach
+    void setUp(){
+        MockitoAnnotations.openMocks(this);
+    }
+
+
+    @Test
+    void getAllTickets_PositiveCase() {
```
Comment (Alexmansar) [inline src/test/java/com/andersen/course/service/TicketsServiceTest.java:37]: don't use __PositiveCase,_Negative and so on
just add some method name like
getAllTickets_ticketsArePresent
or smth loke this

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Tolyaspit Hi,please check small comments

about your questions - for tests is ok to use duplicates in some cases

File: src/test/java/com/andersen/course/service/TicketsServiceTest.java (hunk 2 above)
Comment (Tolyaspit) [reply src/test/java/com/andersen/course/service/TicketsServiceTest.java]: Ok , got it)

**Review by Alexmansar — APPROVED** [review]
@Tolyaspit Hoi)) good job)) could be merged))

**Pull request merged**